README.md

```diff
@@ -1,76 +1,85 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# My Library.
 
-> One paragraph statement about the project.
+> A project where you can create and destroy cards about books.
 
-![screenshot](./app_screenshot.png)
+<br/>
 
-Additional description about the project and its features.
+This project is part of the Microverse curriculum "Javascript." It is made with learning proposes. To practice the creation of objects with constructors and manage basic data from the DOM.
+<br/>
+Here the user can create cards about books with a simple form. Adding Title, the number of pages, and status of reading.
+<br/>
+It is also possible to delete the created cards. The project was created using HTML, CSS, Javascript, and the API, LocalStorage.
 
-## Built With
+<p align="center">
+  <img src="app_screenshot.png" width="800">
+</p>
+
+### Prerequisites
 
-- Major languages
-- Frameworks
-- Technologies used
+- VsCode
+- Access to a Terminal (and basic knowledge on the usage)
+- GitHub account
 
-## Live Demo
+## Download this project!
 
-[Live Demo Link](https://livedemo.com)
+- You will need a [GitHub](https://github.com/) account. 
 
+- Once you have this, at first, you have to clone the repository, [here](https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/cloning-a-repository) you can find out how to make this possible.
 
-## Getting Started
+- Now that you have the repository in your local machine. Change the direction of the local file with your terminal. Just type "$ cd" (without the $ sign) and the path to reach the library-js folder.
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+       $ cd User/Documents/ibrary-js
 
+- You are done! Now just type "$ code ." and to open the folder in the your VsCode.
```

Comment:
- [ ] Please make sure that your README file is descriptive enough for future viewers. For example: How to set up the project locally:-
It's not required for them to have GitHub accounts to clone the repo right?
_It'd be nice if you type instead: `Open your terminal` and it'd be even nicer to put a link to the [`terminal`](https://www.ionos.com/help/email/troubleshooting-mail-basicmail-business/access-the-command-prompt-or-terminal/#:~:text=Open%20Command%20Prompt%20in%20Windows,cmd%22%20and%20then%20click%20OK.)._

- [ ] The next step is to run the CLI `git clone <provide your repo's link here>` in the `terminal`.
Then, `cd <project dir>`, and finally ask them to open the `index.html` file to run the project.

**_Kindly keep in mind that your README file needs to be human-readable because of the non-technicals viewers.
Give yourself a quick reminder with this [template](https://github.com/microverseinc/readme-template)_**